core/aws_ddk_core/stages/glue_transform.py

```diff
@@ -114,16 +107,32 @@ def __init__(
             )
         )
 
-    @property
```

Comment:
I realize that one would be able to use `get_state_machine`, but unless you are strongly opposed to it for some reason, I would want to keep this convenient property to stay consistent with we access entities in our DDK stages

Reply:
Agree, I moved the property to parent class and replaced getter